test/libsolidity/GasMeter.cpp

```diff
@@ -73,9 +73,14 @@ class GasMeterTestFramework: public SolidityExecutionFramework
 		// costs for transaction
 		gas += gasForTransaction(m_compiler.object(m_compiler.lastContractName()).bytecode, true);
 
-		BOOST_REQUIRE(!gas.isInfinite);
-		BOOST_CHECK_LE(m_gasUsed, gas.value);
-		BOOST_CHECK_LE(gas.value - _tolerance, m_gasUsed);
+		// Skip the tests when we force ABIEncoderV2. We should enable this again
```

Comment:
Can you make the "we should enable" part a next line and prepend it with `TODO:` or something similar?